README.md

```diff
@@ -8,4 +8,4 @@ Repository ini digunakan sebagai sayembara pencarian volunter untuk pembuatan ap
 
 Berikut daftar Volunter yang diterima:
 
-* Sindy Chairunisa, [Linkedin](https://www.linkedin.com/in/sindy-chairunisa-b8716418a/).
+* Oon Arfiandwi, [oo.or.id](https://oo.or.id).
```

Comment:
salah, coba benerin lagi. ketik ulang kayak gini

Sindy Chairunisa, [Linkedin](https://www.linkedin.com/in/sindy-chairunisa-b8716418a/).